install/jakartaee/bin/ts.jte

```diff
@@ -1581,56 +1566,30 @@ porting.ts.deploy.class.1=com.sun.ts.lib.implementation.sun.javaee.glassfish.Aut
-# is set to com.sun.ts.lib.deliverable.cts.deploy.StandardDeployment14.
+# @deploy.delay.in.minutes is no longer used, it was for pruned Jakarta Deployment
 ########################################################################
-deploy.delay.in.minutes=5
 
 ########################################################################
-## Vendor's Java EE implementation settings for JSR 88.  These properties
-## are only used when porting.ts.deploy.class.1 is set to
-## com.sun.ts.lib.deliverable.cts.deploy.StandardDeployment14.  These
-## properties are set up to work against SJSAS 9.x PE by default.
-##
-## If you are testing against SJSAS 9.x SE/EE with SSL enabled by default, 
-## you must uncomment the deployManageruri.1 property below that ends with
-## "https", and comment the one that doesn't.
-# 
-# @deployManagerJarFile.1  Location of the jar file that contains the
-#                          JSR 88 deployment manager factory class name
-#                          for the Vendor's Implementation (VI)
-# @deployManageruri.1  JSR 88 management URL, location of the deployment
-#                      process for the VI
-# @deployManageruname.1  username for deployment process for the VI
-# @deployManagerpasswd.1  password for deployment process for the VI
+## Following are no longer used, were settings for pruned Jakarta Deployment 
 ########################################################################
-deployManagerJarFile.1=${javaee.home}/lib/deployment/sun-as-jsr88-dm.jar
-deployManageruri.1=deployer:Sun:AppServer::${deployment_host.1}:${s1as.admin.port}
-#deployManageruri.1=deployer:Sun:AppServer::${deployment_host.1}:${s1as.admin.port}:https
-deployManageruname.1=${s1as.admin.user}
-deployManagerpasswd.1=${s1as.admin.passwd}
+#deployManagerJarFile.1
+#deployManageruri.1
+#deployManageruname.1
+#deployManagerpasswd.1
 
 ########################################################################
-## Java EE RI Settings for JSR 88.  These properties
-## are only used when porting.ts.deploy.class.2 is set to
-## com.sun.ts.lib.deliverable.cts.deploy.StandardDeployment14.
-#
-# @deployManagerJarFile.2  Location of the jar file that contains the
-#                          JSR 88 deployment manager factory class name
-#                          for the Reference Implementation (RI)
-# @deployManageruri.2  JSR 88 management URL, location of the deployment
-#                      process for the RI
-# @deployManageruname.2  username for deployment process for the RI
-# @deployManagerpasswd.2  password for deployment process for the RI
+## Following are no longer used, were settings for pruned Jakarta Deployment
 ########################################################################
-deployManagerJarFile.2=${javaee.home.ri}/lib/deployment/sun-as-jsr88-dm.jar
-deployManageruri.2=deployer:Sun:AppServer::${deployment_host.2}:${ri.admin.port}
-deployManageruname.2=${ri.admin.user}
-deployManagerpasswd.2=${ri.admin.passwd}
+#deployManagerJarFile.2
```

Comment:
Property fetching causing exception:

As of now there are exceptions while sample test run as below :
" com.sun.ts.lib.deliverable.PropertyNotSetException: No value found for deployManagerJarFile.1.  Please check your jte file for an appropiate value " from ./src/com/sun/ts/lib/deliverable/AbstractPropertyManager.java
So looks like it does not allow null for these currently when fetched at src/com/sun/ts/lib/deliverable/cts/CTSPropertyManager.java

Reply:
Thanks @alwin-joseph, I will look at this.

Reply:
Push some possible changes to https://github.com/scottmarlow/jakartaee-tck/tree/moreDeploymentRemoval

Reply:
We are now getting ```execStatus=Failed. Illegal access to test\: java.lang.NullPointerException```

Reply:
@alwin-joseph from the ./root/vi/glassfish6/glassfish/domains/domain1/logs/server.log from [samples-results.tar.gz](https://ci.eclipse.org/jakartaee-tck/job/jakartaee-tck/job/master/678/artifact/samples-results.tar.gz):
```
[2020-06-08T17:09:01.779+0000] [glassfish 6.0] [INFO] [NCLS-DEPLOYMENT-02027] [javax.enterprise.system.tools.deployment.autodeploy] [tid: _ThreadID=54 _ThreadName=AutoDeployer] [timeMillis: 1591636141779] [levelValue: 800] [[
  Selecting file /root/vi/glassfish6/glassfish/domains/domain1/autodeploy/ejb_sam_Hello.ear for autodeployment]]
...
[2020-06-08T17:09:03.332+0000] [glassfish 6.0] [SEVERE] [NCLS-CORE-00029] [javax.enterprise.system.core] [tid: _ThreadID=54 _ThreadName=AutoDeployer] [timeMillis: 1591636143332] [levelValue: 1000] [[
  Cannot start container ejb, exception: A MultiException has 7 exceptions.  They are:
1. com.sun.enterprise.module.ResolveError: Failed to start OSGiModuleImpl:: Bundle = [org.glassfish.main.orb.iiop [218]], State = [NEW]
2. java.lang.IllegalStateException: Could not load descriptor SystemDescriptor(
        implementation=org.glassfish.enterprise.iiop.impl.GlassFishORBFactoryImpl
        contracts={org.glassfish.enterprise.iiop.impl.GlassFishORBFactoryImpl,org.glassfish.enterprise.iiop.api.GlassFishORBFactory}
        scope=jakarta.inject.Singleton
        qualifiers={}
        descriptorType=CLASS
        descriptorVisibility=NORMAL
        metadata=Bundle-SymbolicName={org.glassfish.main.orb.iiop},Bundle-Version={6.0.0.SNAPSHOT}
        rank=0
        loader=OsgiPopulatorPostProcessor.HK2Loader(OSGiModuleImpl:: Bundle = [org.glassfish.main.orb.iiop [218]], State = [NEW],655682992)
        proxiable=null
        proxyForSameScope=null
        analysisName=null
        id=110
        locatorId=0
        identityHashCode=444705614
        reified=false)
3. java.lang.IllegalStateException: Unable to perform operation: post construct on org.glassfish.enterprise.iiop.api.GlassFishORBHelper
4. java.lang.IllegalArgumentException: While attempting to resolve the dependencies of com.sun.ejb.containers.EjbContainerUtilImpl errors were found
5. java.lang.IllegalStateException: Unable to perform operation: resolve on com.sun.ejb.containers.EjbContainerUtilImpl
6. java.lang.IllegalArgumentException: While attempting to resolve the dependencies of org.glassfish.ejb.startup.EjbContainerStarter errors were found
7. java.lang.IllegalStateException: Unable to perform operation: resolve on org.glassfish.ejb.startup.EjbContainerStarter
```
NPE call stack is
```
[runcts] OUT => [javatest.batch] java.lang.NullPointerException
   [runcts] OUT => [javatest.batch]     at com.sun.ts.lib.harness.TSScript.run(TSScript.java:320)
   [runcts] OUT => [javatest.batch]     at com.sun.javatest.Script.run(Script.java:241)
   [runcts] OUT => [javatest.batch]     at com.sun.javatest.DefaultTestRunner.runTest(DefaultTestRunner.java:174)
   [runcts] OUT => [javatest.batch]     at com.sun.javatest.DefaultTestRunner.access$100(DefaultTestRunner.java:43)
   [runcts] OUT => [javatest.batch]     at com.sun.javatest.DefaultTestRunner$1.run(DefaultTestRunner.java:66)
```

Reply:
Created minor NPE fix via https://github.com/eclipse-ee4j/jakartaee-tck/pull/311.

Reply:
> Pushed some possible changes to https://github.com/scottmarlow/jakartaee-tck/tree/moreDeploymentRemoval

This issue is fixed now in the latest build. Thanks.

Reply:
> Created minor NPE fix via #311.

Scott - From what I see the root problem is that the deployment fails due to some reason. I am not sure if the fix is addressing that. The deployment failures are there for several packages even for standalone tck runs and need to be looked into. We can use the tck bundles built and try them in local to check the server logs. It is also possible something is wrong with the deployment rar/war or with glassfish itself.

Reply:
From the server logs :

[2020-06-08T16:58:42.522+0000] [glassfish 6.0] [WARNING] [] [] [tid: _ThreadID=52 _ThreadName=AutoDeployer] [timeMillis: 1591635522522] [levelValue: 900] [[
  Error occurred
org.xml.sax.SAXParseException; lineNumber: 52; columnNumber: 106; Deployment descriptor file META-INF/ra.xml in archive [whitebox-tx.rar].  cvc-enumeration-valid: Value 'jakarta.resource.spi.security.PasswordCredential' is not facet-valid with respect to enumeration '[javax.resource.spi.security.PasswordCredential, org.ietf.jgss.GSSCredential, javax.resource.spi.security.GenericCredential]'. It must be a value from the enumeration.

I see glassfish6/glassfish/lib/schemas/connector_1_7.xsd & glassfish6/glassfish/lib/dtds/connector_1_5.dtd having javax.resource.spi.security.PasswordCredential. I suspect these are to be changed to jakarta.* in glassfish ?

Reply:
Good catch, yes, I agree that glassfish should update these.
I did find https://github.com/eclipse-ee4j/glassfish/issues/23080 which is for the ORB failure, that we also need fixed I think.

Links for updated XSDs:
https://jakarta.ee/xml/ns/jakartaee/connector_2_0.xsd
https://github.com/eclipse-ee4j/jakartaee-schemas/blob/master/jakartaee9/src/connector_2_0.xsds

It looks like we already updated jakartaee-tck/src/com/sun/ts/tests/common/connector/whitebox/permissiondd/ra.xml, which is good.

I'm not sure if any of the TCK lib/schemas are actually used by the Platform TCK but we should update any of those that are needed, to use their EE 9 equivalents (as per my discussion with legal@eclipse.org).  I think the same for lib/dtds.